src/class-parsely.php

```diff
@@ -653,7 +655,9 @@ public static function get_asset_cache_buster(): string {
 		 *
 		 * @param string $cache_buster Plugin version, unless WP_DEBUG is defined and truthy, and tests are not running.
 		 */
-		return apply_filters( 'wp_parsely_cache_buster', (string) $cache_buster );
+		$cache_buster = apply_filters( 'wp_parsely_cache_buster', (string) $cache_buster );
+
+		return $cache_buster;
```

Comment:
This is a fix for a bug that would result in inconsistent behavior if anyone has this `wp_parsely_cache_buster ` filter set.

Namely, the static variable would be set above, the result filtered once and returned, & the unfiltered version used on subsequent calls.  

Since this change deprecates the function altogether, this filter should also be deprecated, but I wasn't sure if we should fix the bug or leave it as is.  Thoughts?

Reply:
Use `apply_filters_deprecated()`.